docs/doxygen/platforms/msw_install.md

```diff
@@ -46,22 +37,22 @@ actually required, this makes using the library more convenient and
 this environment variable is used in the examples below.
 
 NB: If you checked your sources from version control repository and
-    didn't obtain them from a release file, you also need to copy
-    include/wx/msw/setup0.h to include/wx/msw/setup.h and to remember
-    to update the latter whenever the former changes, otherwise you
-    will get compilation errors if any new symbols are added to
-    setup0.h file in the repository.
+didn't obtain them from a release file, you also need to copy
+*include/wx/msw/setup0.h* to *include/wx/msw/setup.h* and to remember
```

Comment:
Minor, but I think we should use backticks instead of asterisks around the file names.